test/Integration/ChainRules_test.jl

```diff
@@ -68,6 +67,14 @@
             test_rrule(contract, a, b)
         end
 
+        @testset "[matrix multiplication, complex numbers]" begin
+            a = Tensor(rand(Complex{Float64}, 2, 2), (:i, :k))
+            b = Tensor(rand(Complex{Float64}, 2, 2), (:k, :j))
+
+            test_frule(contract, a, b)
+            test_rrule(contract, a, b)
+        end
+
```

Comment:
Put this code inside the `["matrix multiplication"]` testset. You can nest testsets inside testsets.

Reply:
Okay, I will add it now.